codemods/migrate-text/data.ts

```diff
@@ -0,0 +1,16 @@
+export const textComponentMap = {
+    D1: "HeaderXXL",
+    D2: "HeaderXL",
+    D3: "HeaderMD",
+    D4: "HeaderSM",
+    H1: "HeaderLG",
+    H2: "HeaderMD",
+    H3: "HeaderSM",
+    H4: "HeaderXS",
+    H5: "HeaderXS",
+    H6: "HeaderXS",
```

Comment:
```suggestion
    H5: "BodyMD",
    H6: "BodySM",
```

Reply:
Hi Rou Ling would changing the H5 to Body affect the consumer intended tag of H5 to P tags?

Reply:
yes, the consumer needs to set the `as` prop accordingly as mentioned in the migration docs